src/ICSharpCode.SharpZipLib/Tar/TarArchive.cs

```diff
@@ -356,8 +356,7 @@ public string RootPath
 				{
 					throw new ObjectDisposedException("TarArchive");
 				}
-				// Convert to forward slashes for matching. Trim trailing / for correct final path
-				rootPath = value.Replace('\\', '/').TrimEnd('/');
+				rootPath = value.ClearTarPath().TrimEnd('/');
```

Comment:
```suggestion
				rootPath = value.ToTarArchivePath().TrimEnd('/');
```

Reply:
Thanks a lot for your suggestions. They do absolutely make sense. Just out of curiosity. Didn't #593 address  this issue already and this PR is thus obsolete now?